Firmware/mmu2.cpp

```diff
@@ -484,6 +484,7 @@ void MMU2::ResumeAndUnPark(bool move_axes, bool turn_off_nozzle) {
                 lcd_display_message_fullscreen_P(_i("MMU OK. Resuming temperature...")); // better report the event and let the GUI do its work somewhere else
             });
             LogEchoEvent("Hotend temperature reached");
+            lcd_update_enable(true); // temporary hack to stop this locking the printer...
```

Comment:
Just a note: We should probably include this in `lcd_display_message_fullscreen_P()` in the future. It calls `lcd_update_enable(false)` but it doesn't set it back to `true` when used like in line 484.

Reply:
I looked again, and you're right 👍